src/cpu/flexcpu/inflight_inst.cc

```diff
@@ -30,6 +30,8 @@
 
 #include "cpu/flexcpu/inflight_inst.hh"
 
+#include <algorithm>
```

Comment:
I could be missing this, but I don't see where it's used.

Reply:
Oh right, I added it to use find, but it turned out I didn't use it I think